### PR TITLE
License editor autosize

### PR DESCRIPTION
**Pull request by cscwooller**

### CODE REVIEW

File: src/cljs/rems/administration/components.cljs [hunk 1]
```diff
@@ -28,20 +28,28 @@
...
-     [:input.form-control {:type "text"
+     [:input.form-control {:type type
                            :id id
                            :placeholder placeholder
-                           :value (get-in form keys)
+                           :value     (get-in form keys)
                            :on-change #(rf/dispatch [(:update-form context)
-                                                     keys
-                                                     (.. % -target -value)])}]]))
+                                 keys
```
[the remark this answers is not in the document]
Comment (cscwooller) [reply src/cljs/rems/administration/components.cljs]: Fixed and pushed

File: src/cljs/rems/administration/components.cljs [hunk 2]
```diff
@@ -28,20 +28,28 @@
        (map key-to-id)
        (str/join "-")))
 
-(defn text-field
-  "A basic text field, full page width."
-  [context {:keys [keys label placeholder]}]
+(defn input-field [type context {:keys [keys label placeholder]}]
```
[the remark this answers is not in the document]
Comment (cscwooller) [reply src/cljs/rems/administration/components.cljs]: I would like to keep at least type maybe separate?

[the remark this answers is not in the document]
Comment (Macroz) [reply src/cljs/rems/administration/components.cljs]: This is what I mean https://github.com/CSCfi/rems/blob/171d2d8c15df4aa902a4c9f7ae0e2897297d88ec/src/cljs/rems/collapsible.cljs#L61

[the remark this answers is not in the document]
Comment (cscwooller) [reply src/cljs/rems/administration/components.cljs]: Pushed now.

**Pull request merged**